lab3react/src/inCloudApp.js

```diff
@@ -0,0 +1,237 @@
+import App from './App.js';
+import {useState} from "react";
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+import firebase from "firebase/compat";
+import {useCollection} from "react-firebase-hooks/firestore";
+
+const firebaseConfig = {
+    apiKey: "AIzaSyCd9qqxvMpEKpBzwfWcc2tlRFa6ICaLH_s",
+    authDomain: "hmc-cs124-fa21-labs.firebaseapp.com",
+    projectId: "hmc-cs124-fa21-labs",
+    storageBucket: "hmc-cs124-fa21-labs.appspot.com",
+    messagingSenderId: "949410042946",
+    appId: "1:949410042946:web:0113b139a7e3cd1cc709db"
+};
+firebase.initializeApp(firebaseConfig);
+const db = firebase.firestore();
+const collectionName = "acowe-tasks-2"
+
+
+function InCloudApp(props) {
+    const query = db.collection(collectionName);
+    const [value, loading, error] = useCollection(query);
+    const [currentList, setCurrentList] = useState("wow");
+
+    const [sortVal, setSortVal] = useState("default");
+    const [sortPriority, setSortPriority] = useState("place_order");
+    const [sortDirection, setSortDirection] = useState('asc');
+
+    const query_2 = db.collection(collectionName).doc(currentList)
+        .collection(currentList + "_tasks").orderBy(sortPriority,sortDirection);
```

Comment:
Just use tasks for the subcollection name. No need to make unique with currentList.